package.json

```diff
@@ -53,5 +53,8 @@
       "module",
       "typescript"
     ]
+  },
+  "dependencies": {
+    "svg-path-properties": "^0.4.8"
```

Comment:
I'm not a big fan of `^` because it can cause unspecified errors. I know about `.lock` files and etc, but in past, I've run in some very specific error related to auto updates and now I update packages manually. Not everyone is following semantic versioning :)

Reply:
I agree 👍🏻